### ISSUE TITLE
Environment variables not used to locate gcc/g++

### ISSUE THREAD

**Issue by kitling** [hebaishi/linter-gcc#82]
## Checklist
* [x] Check that you have the GNU C Compiler installed by typing ```gcc --version``` into a terminal.
* [x] If you don't have the ```linter``` package installed, either update to the latest version of ```linter-gcc``` or install linter directly with ```apm install linter```.
* [x] Ensure that you are using a valid grammar - either C/C++/C++14
* [x] Ensure that you have read all the relevant information in the [wiki](https://github.com/hebaishi/linter-gcc/wiki), particularly the [Common Issues](https://github.com/hebaishi/linter-gcc/wiki/Common-Issues) page.

## Issue (including debugging messages):
In a nutshell, #45 but with a different variable.

It would be useful to have this in the case where you are working on a project with a cross compiler that is not in your path. (E.G. working for the 3ds with devkitARM)

### Workaround:
Add the offending compiler to your path.
